generator/validate/gen.go

```diff
@@ -34,11 +34,7 @@ func (Generator) RegisterMarkers(into *markers.Registry) error {
 // Generate validates the source code
 func (g Generator) Generate(ctx *genall.GenerationContext) error {
 	for _, root := range ctx.Roots {
-		ctx.Checker.Check(root, func(node ast.Node) bool {
-			// ignore interfaces
-			_, isIface := node.(*ast.InterfaceType)
-			return !isIface
-		})
+		//ctx.Checker.Check(root)
```

Comment:
Is this intentional?

Reply:
Good catch. I did it to fix nil pointer exception before I found the proper way to fix it. Reverting.